specifyweb/frontend/js_src/lib/components/WorkBench/WbSpreadsheet.tsx

```diff
@@ -464,64 +280,10 @@ function WbSpreadsheetComponent({
         stretchH="all"
```

Comment:
```ts
// @ts-expect-error typing error, possibly fixed in v14
```

don't forget to resolve this before merging this PR
if it's complaining about passing `true` to `sortIndicator`, then maybe it's because Hot changed their API and now they want you to pass the prop differently - disabling typescript is not a good option here, unless there is a bug with their typings

Reply:
looks like sortIndicator doesn't exist anymore in their API and is a default behavior.